src/acvm_interop/proof_system.rs

```diff
@@ -5,6 +5,8 @@ use crate::gnark_backend_glue as gnark_backend;
 
 use super::Gnark;
 
+// Allow unwrap because the ProofSystemCompiler interface does not support Result
+#[allow(clippy::unwrap_used)]
```

Comment:
Let's add this allow at the module level (in `src/acvm_interop/mod.rs`). The other traits we're implementing have the same issue with the `unwrap`.

Reply:
👍
I think we should merge #37 first then

Reply:
#37 merged!